pkg/operation/botanist/kubeapiserver_test.go

```diff
@@ -527,29 +528,122 @@ exemptions:
   runtimeClasses: ["random"]
   namespaces: ["random"]
 `),
+									},
 								},
-							},
-						}
+							}
+						})
+
+						It("should add kube-system to exempted namespaces and not touch other fields", func() {
+							var admConfig *admissionapiv1beta1.PodSecurityConfiguration
+
+							Expect(configData).NotTo(BeNil())
+
+							config, err = runtime.Decode(codec, configData.Raw)
+							Expect(err).NotTo(HaveOccurred())
+
+							admConfig, ok = config.(*admissionapiv1beta1.PodSecurityConfiguration)
+							Expect(ok).To(BeTrue())
+
+							Expect(admConfig.Defaults).To(MatchFields(IgnoreExtras, Fields{
+								"Enforce": Equal("privileged"),
+								// This field is defaulted by kubernetes
+								"Audit":       Equal("privileged"),
+								"Warn":        Equal("baseline"),
+								"WarnVersion": Equal("v1.23"),
+							}))
+							Expect(admConfig.Exemptions.Usernames).To(ContainElement("admin"))
+							Expect(admConfig.Exemptions.Namespaces).To(ContainElements("kube-system", "random"))
+						})
 					})
 
-					It("should add kube-system to exempted namespaces and not touch other fields", func() {
-						Expect(configData).NotTo(BeNil())
+					Context("kubernetes version >= 1.22.0", func() {
```

Comment:
```suggestion
					Context("kubernetes version >= 1.22.0 and < 1.23.0", func() {
```
I think it is a little confusing otherwise.